### PR TITLE
Compare time instead of dates

### PR DESCRIPTION
**Pull request by punkeel**
The former code would fail (imo) at 11PMxx: the asleep variable
contains yesterday's 11PM (which is after now, √) but awake would be
today's 9AM, which is not before now -> the && fails

PS/ Contribution to not-useful projects is my favorite hobby 💯

### CODE REVIEW

File: app/do.js [hunk 1]
```diff
@@ -5,18 +5,20 @@ ipc.on('show', function (event, message) {
   doItGetTheTimeDoItNow();
 })
 
+var minutesOfDay = function(m){
+  return m.minutes() + m.hours() * 60;
+}
+
 function doItGetTheTimeDoItNow() {
-  var now = moment().tz("Asia/Taipei");
+  var now = minutesOfDay( )moment().tz("Asia/Taipei"));
```
Comment (notwaldorf) [inline app/do.js]: it looks like there's an extra bracket here?

Comment (punkeel) [reply app/do.js]: You're right

**Comment by notwaldorf** [comment]
@PunKeel Oh thank god, timezone math is my least favourite thing! 😵

Also I think I pushed some things and you need a rebase. Sorry!

**Comment by punkeel** [comment]
@notwaldorf Rebased! 👍 

If I remember correctly, GitHub lets you merge my commits into one, so I won't do it

File: app/do.js [hunk 2]
```diff
@@ -5,18 +5,20 @@ ipc.on('show', function (event, message) {
   doItGetTheTimeDoItNow();
 })
 
+var minutesOfDay = function(m) {
+  return m.minutes() + m.hours() * 60;
+}
+
 function doItGetTheTimeDoItNow() {
-  var now = moment().tz('Asia/Taipei');
+  var now = minutesOfDay(moment().tz('Asia/Taipei'));
```
Comment (notwaldorf) [inline app/do.js]: Oh so this `now` is used later to display the time (with a `now.format()`). I think you need a different variable either here or later.

**Comment by notwaldorf** [comment]
I'll just merge it and fix the string in a follow up PR. Thanks!!

**Comment by punkeel** [comment]
You're welcome! 
Sorry about the multiple fails.

PS/ Isn't the string fixed with this line: https://github.com/notwaldorf/is-mu-an-asleep/pull/1/commits/599e262b9067a1a7dfaa4f058b8b2eacd013cc91#diff-93b3910207774b01d21c9e5b050793f9R14 ? 🤔

**Comment by notwaldorf** [comment]
@PunKeel Yeah! I noticed that after merging it. But then the SVG `innerText` doesn't work (you need `innerHTML`), but it was a quick fix. Yay!

**Comment by punkeel** [comment]
@notwaldorf Worked in chrome, I assumed it would work on Electron too. My bad.
(I'm still trying to download Electron, slow connection doesn't help)

Yay! \o/

**Pull request merged**